api/tests/tooling/database-builder/factory/build-competence-mark.js

```diff
@@ -0,0 +1,24 @@
+const faker = require('faker');
+const buildAssessmentResult = require('./build-assessment-result');
+const databaseBuffer = require('../database-buffer');
+
+module.exports = function buildCompetenceMark({
+  id = faker.random.number(),
+  level = faker.random.number(),
+  score = faker.random.number(),
+  area_code = faker.random.number(),
+  competence_code = `${faker.random.number()}_${faker.random.number()}`,
+  assessmentResultId = buildAssessmentResult().id,
```

Comment:
Je crois que @Radioreve a levé un problème avec cette manière de faire dans :
https://github.com/1024pix/pix/pull/470

Reply:
C'était quoi déjà ? Le problème est-il corrigé dans la PR qui fixe le `bookshelf-to-domain-converter` ?

Reply:
Effectivement

Reply:
updated